src/test/java/org/opensearch/security/IntegrationTests.java

```diff
@@ -331,7 +331,7 @@ public void testSpecialUsernames() throws Exception {
         setup();
         RestHelper rh = nonSslRestHelper();
 
-        Assert.assertEquals(HttpStatus.SC_OK, rh.executeGetRequest("", encodeBasicHeader("bug.99", "nagilum")).getStatusCode());
```

Comment:
This bug.99 was a service account btw, so we have to replace it with a non-service account, since service account is not allowed anymore